lxd/network/openvswitch/ovs.go

```diff
@@ -373,3 +374,151 @@ func (o *OVS) OVNSouthboundDBRemoteAddress() (string, error) {
 
 	return addr, nil
 }
+
+// Default STP priority is 32768 (0x8000). This value will be used if the
+// priority cannot be retrieved or if STP is disabled
+func GetStpPriority(bridgeName string) (string, error) {
+	output, err := shared.RunCommand("ovs-appctl", "stp/show", bridgeName)
+	if err != nil {
+		runErr, ok := err.(shared.RunError)
+		if ok {
+			exitError, ok := runErr.Unwrap().(*exec.ExitError)
+
+			// STP is disabled for that bridge
+			if ok && exitError.ExitCode() == 2 {
+				return "8000", nil
+			}
+		}
+		return "8000", err
```

Comment:
You will always need a newline after a closing brace and before any subsequent statements.